src/gui/RaceView.java

```diff
@@ -85,6 +87,7 @@ public void reloadGui() {
             model.removeRow(0);
         }
         boolean isFirstRow = true;
+        boolean isSecondRow = false;
```

Comment:
kannsch du die variable nid wägloh und de index vom loop verwende?

Reply:
you're right, das isch schlauer